flink-ml-core/src/test/java/org/apache/flink/ml/api/StageTest.java

```diff
@@ -383,6 +398,7 @@ public void testStageSaveLoad() throws IOException {
         stage.set(MyParams.DOUBLE_ARRAY_PARAM, new Double[] {50.0, 51.0});
         stage.set(MyParams.STRING_ARRAY_PARAM, new String[] {"50", "51"});
         stage.set(MyParams.VECTOR_PARAM, Vectors.dense(2, 3, 4));
+        stage.set(MyParams.WINDOWS_PARAM, EventTimeSessionWindows.withGap(Time.milliseconds(100)));
```

Comment:
Would it be useful to also validate that other Windows subclasses can be serialized and de-serialized correctly?